oct MF release notes and jdbc (#6377)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-cloud-apis/sl-jdbc.md

}}
```

</Expandable>

<Expandable alt_header="Fetch metrics by substring search">

> You can filter your metrics to include only those that contain a specific substring. Use the `search` argument to specify the substring you want to match.

Reviewer: ```suggestion
You can filter your metrics to include only those that contain a specific [substring](/reference/dbt-jinja-functions/cross-database-macros#position). Use the `search` argument to specify the substring you want to match.
```

Reviewer: Hiya @mirnawong1 

This is a non-blocking suggestion but I thought it might be a good idea to link out the word substring in case the user doesn't know what it is.

Kind Regards
Natalie

Author: hey @nataliefiann , that links to a jinja macro, 'position' so i'm not sure its a good link. since it's a release note, i don't think we need we need to spell it out but i can add that it's a text in the jdbc doc

> You can filter your metrics to include only those that contain a specific substring. Use the `search` argument to specify the substring you want to match.

Author: ```suggestion
You can filter your metrics to include only those that contain a specific substring (sequence of characters contained within a larger string (text)). Use the `search` argument to specify the substring you want to match.
```

```sql
select * from {{ semantic_layer.metrics(search='order') }}
```

If no substring is provided, the query returns all metrics.

</Expandable> 

<Expandable alt_header="Paginate metadata calls">

> In cases where user manifests are large, pagination is a useful way to prevent query character limits in the data platform. You can paginate results for both `semantic_layer.metrics()` and `semantic_layer.dimensions()` calls using the `page_size` and `page_number` parameters.

Reviewer 2: "In cases where user manifests are large, pagination is a useful way to prevent query character limits in the data platform. You can paginate results" 

Change to: "In the case when you don't want to return the full result set from a metadata call you can paginate the resluts.."

> In cases where user manifests are large, pagination is a useful way to prevent query character limits in the data platform. You can paginate results for both `semantic_layer.metrics()` and `semantic_layer.dimensions()` calls using the `page_size` and `page_number` parameters.

Author: ```suggestion
In the case when you don't want to return the full result set from a metadata call, you can paginate the results for both `semantic_layer.metrics()` and `semantic_layer.dimensions()` calls using the `page_size` and `page_number` parameters.
```

> - `page_size`: Optional, sets the number of records per page. If left as `None`, there is no page limit.

Reviewer: ```suggestion
- `page_size`: This is an optional feature which sets the number of records per page. If left as `None`, there is no page limit.
```

> - `page_size`: This is an optional feature which sets the number of records per page. If left as `None`, there is no page limit.

Author: ```suggestion
- `page_size`: This is an optional variable which sets the number of records per page. If left as None, there is no page limit.
```

> - `page_size`: This is an optional feature which sets the number of records per page. If left as `None`, there is no page limit.

Reviewer 2: Change to --> This is an optional variable which sets the number of records per page. If left as `None`, there is no page limit.

> - `page_number`: Optional, specifies the page number to retrieve. Defaults to `1` (first page) if not specified.

Reviewer: ```suggestion
- `page_number`: This is another optional feature which specifies the page number to retrieve. Defaults to `1` (first page) if not specified.
```

Reviewer: Hiya @mirnawong1 

I added this non-blocking suggestion as I thought it might flow better as a full sentence.

Kind Regards
Natalie

> - `page_number`: This is another optional feature which specifies the page number to retrieve. Defaults to `1` (first page) if not specified.

Author: ```suggestion
- `page_number`: This is an optional variable which specifies the page number to retrieve. Defaults to `1` (first page) if not specified.
```

> - `page_number`: This is another optional feature which specifies the page number to retrieve. Defaults to `1` (first page) if not specified.

Reviewer 2: Same as above
